platform/darwin/app/LimeGreenStyleLayer.m

```diff
@@ -1,7 +1,12 @@
 #import "LimeGreenStyleLayer.h"
 
-#include <OpenGLES/ES2/gl.h>
-#include <OpenGLES/ES2/glext.h>
+#if TARGET_OS_IPHONE
```

Comment:
If we were to instead `@import GLKit;` here, that would allow us to get rid of these conditionals and remove `OpenGLES.framework` as an explicitly linked framework in iosapp.